docs/user/pixel_observations.md

```diff
@@ -55,13 +55,13 @@ working with pixel observations. You may use multiple wrappers simultaneously,
 though the order in which you use them matters. See the pixel-specific wrappers
 section for more details on this.
 
-### GarageEnv
+### GymEnv
```

Comment:
Isn't much of this true of all of `Environment`, including the `is_image` part?

Reply:
`PointEnv`, for example, doesn't take `is_image`. Docs page can be updated later after this PR?